resources/views/contests/show.blade.php

```diff
@@ -27,6 +27,9 @@
                     <li>
                         <a href="{{ route('split', $contest->waitingRoom->id) }}" class="button -secondary">Split</a>
                     </li>
+                    <li>
+                        <a href="{{ route('contests.user.add', $contest->id) }}" class="button -secondary">+ User</a>
```

Comment:
i had that first and it ruins everything
![image](https://cloud.githubusercontent.com/assets/645205/14956707/23bce188-104f-11e6-88d7-52bd1e411d74.png)